### ISSUE TITLE
Automatic Nightly Backups

### ISSUE THREAD

**Issue by markwaters** [redfish64/TinyTravelTracker#14]
Firstly , I have been using Tiny Travel Tracker for about a month and love it!
Thank you so much for writing it.

My phone got wet yesterday and has died , the latest backup I have is from about 5 days ago.

Could backups be automated ?

**Comment by xloem** [comment]
I agree, it would be great to be able to at least save the backup to sdcard every night

**Comment by ilf** [comment]
I think this is the job of a backup app like Titanium Backup.

**Comment by xloem** [comment]
I'm interested in automatically uploading the gps data elsewhere.  At the moment it is stuck inside the app.

**Comment by ilf** [comment]
The data is not "stuck", it's in /sdcard/Android/data/com.rareventure.gps2/files/gps.db3

**Comment by xloem** [comment]
Thanks, that's really helpful.

It's not really a replacement for the easily-readable XML file though.  Even running the db3 through sqlite the fields are still binary blobs.

**Comment by redfish64** [comment]
Although that file on the sdcard contains the points, without the private
key, which is stored on the internal flash disk, it cannot be decrypted.
You either need to backup the points through the settings menu in the app,
or backup both the external memory, and the internal settings (which holds
the key).

I believe Titanium backup would work for the latter but I have never used
it, myself.
